client/web/views/root_view/add_access_key_dialog/index.ts

```diff
@@ -174,7 +174,7 @@ export class AddAccessKeyDialog extends LitElement {
     }
   }
 
-  private edit(event: InputEvent) {
+  private async edit(event: InputEvent) {
```

Comment:
No need for this change.

```suggestion
  private edit(event: InputEvent) {
```